app/controllers/assessments_controller.rb

```diff
@@ -1,5 +1,14 @@
 # frozen_string_literal: true
 
+# For backwards-compatibility reasons, we still want to support the old 2-letter language codes
+LEGACY_LANGUAGE_MAPPING = {
```

Comment:
I think this should at least be defined within a class

Reply:
Done within a module, per @Bialogs

Reply:
Done!